src/account/components/AccountLogin.js

```diff
@@ -39,6 +40,11 @@ import logo from 'assets/logo.svg';
 
 import theme from 'theme';
 
+// ref: https://mui.com/material-ui/icons/#svgicon
+const MicrosoftIcon = props => {
+  return <SvgIcon {...props}><path d="M2 3h9v9H2V3m9 19H2v-9h9v9M21 3v9h-9V3h9m0 19h-9v-9h9v9Z"/></SvgIcon>;
```

Comment:
Ideally the svg would be stored in a `.svg` file and somehow loaded here when it's imported, but I couldn't get this to work. Anyone have any ideas how to do this?

Reply:
I found a suggestion here: https://stackoverflow.com/questions/60136470/how-to-assign-a-custom-icon-to-materialui-buttons-starticon

Reply:
please rename the file to `Microsoft.svg`.